easybuild/easyconfigs/o/Octave/Octave-5.1.0-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,58 @@
+# @author: gppezzi (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'Octave'
+version = '5.1.0'
+
+homepage = 'http://www.gnu.org/software/octave/'
+description = "GNU Octave is a high-level interpreted language, primarily intended for numerical computations."
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'pic': True, 'dynamic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('Bison', '3.3.2'),
```

Comment:
@omlins What about old dependencies that I have found in the previous recipes? E.g. `texinfo`